packages/edit-site/src/components/sidebar-dataviews/dataview-item.js

```diff
@@ -13,7 +13,7 @@ import { unlock } from '../../lock-unlock';
 const { useLocation } = unlock( routerPrivateApis );
 
 function getDataViewIcon( type ) {
-	const icons = { list: page, grid: columns };
+	const icons = { list: page, grid: columns, 'side-by-side': mediaAndText };
```

Comment:
What do you think about using `pullRight` instead? What I like about mediaText is that it conveys more of a list (3 lines instead of two in pullRight). What I like about pullRight is that it's in the proper order of the layout (left is list, right is frame).

<img width="247" alt="Captura de ecrã 2023-11-07, às 13 39 03" src="https://github.com/WordPress/gutenberg/assets/583546/d03b7ea0-c689-4858-b9ee-4905493d4eb4">

<img width="247" alt="Captura de ecrã 2023-11-07, às 13 39 10" src="https://github.com/WordPress/gutenberg/assets/583546/089c6d19-07d5-47c5-b954-679d564e98ab">